.github/workflows/gpu-ci.yml

```diff
@@ -18,38 +18,41 @@ jobs:
       run: |
         rm -f ~/.dace.conf
         rm -rf .dacecache tests/.dacecache
-        . /opt/setupenv
+        python -m venv ~/.venv      # create venv so we can use pip
+        source ~/.venv/bin/activate # activate venv
         python -m pip install --upgrade pip
         pip install flake8 pytest-xdist coverage
         pip install mpi4py
+        pip install cupy
```

Comment:
Wouldn't it be better to keep the dependencies updating over time? For most runs it will detect that it is already installed in the most up to date version.

Reply:
Sure. Sometimes cupy historically had issues to compile, but if it works it works.